endpoints/openrtb2/video_auction.go

```diff
@@ -514,18 +562,37 @@ func (deps *endpointDeps) validateVideoRequest(req *openrtb_ext.BidRequestVideo)
 		err := errors.New("request missing required field: PodConfig.Pods")
 		errL = append(errL, err)
 	}
+	podErrors := make([]PodError, 0, 0)
+	podIdsSet := make(map[int]bool)
 	for ind, pod := range req.PodConfig.Pods {
+		podErr := PodError{}
+
+		if podIdsSet[pod.PodId] == true {
+			err := fmt.Sprintf("request duplicated required field: PodConfig.Pods.PodId, Pod id: %d", pod.PodId)
+			podErr.PodErrors = append(podErr.PodErrors, err)
+		} else {
+			podIdsSet[pod.PodId] = true
+		}
 		if pod.PodId <= 0 {
-			err := fmt.Errorf("request missing required field: PodConfig.Pods.PodId, Pod index: %d", ind)
-			errL = append(errL, err)
+			err := fmt.Sprintf("request missing required field: PodConfig.Pods.PodId, Pod index: %d", ind)
+			podErr.PodErrors = append(podErr.PodErrors, err)
 		}
 		if pod.AdPodDurationSec == 0 {
-			err := fmt.Errorf("request missing required field: PodConfig.Pods.AdPodDurationSec, Pod index: %d", ind)
-			errL = append(errL, err)
+			err := fmt.Sprintf("request missing required field: PodConfig.Pods.AdPodDurationSec, Pod index: %d", ind)
+			podErr.PodErrors = append(podErr.PodErrors, err)
```

Comment:
I am assuming it is possible that someone set the ad pod duration in the request as `0`? In that case we should probably modify the error message to capture both possibilities of ad pod duration being missing or being set to '0'.

Reply:
When AdPodDurationSec is missed in request, after unmarshalling it to object we have zero as default value. This is the same if it is set to zero initially. there is no way to distinguish it in code. 
I changed error message to this:
```request missing or incorrect required field: PodConfig.Pods.AdPodDurationSec, Pod index: %d```

Reply:
ConfigId as well

Reply:
@mansinahar Good comments, thank you!

Reply:
Yeah, exactly. It's hard to figure out if the `0` was set by the user or is just the default value. Thanks for updating the error message to capture both possibilities.